<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DicomSender</class>
 <widget class="QMainWindow" name="DicomSender">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>400</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>625</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dicom Sender</string>
  </property>
  <property name="windowIcon">
   <iconset resource="DicomSender.qrc">
    <normaloff>:/icons/send.svg</normaloff>:/icons/send.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="dirsList">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>500</width>
             <height>26</height>
            </size>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonOpen">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>28</height>
            </size>
           </property>
           <property name="text">
            <string>Open</string>
           </property>
           <property name="icon">
            <iconset resource="DicomSender.qrc">
             <normaloff>:/icons/open.svg</normaloff>:/icons/open.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="4">
            <widget class="QPushButton" name="buttonSend">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>28</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>28</height>
              </size>
             </property>
             <property name="text">
              <string>Send</string>
             </property>
             <property name="icon">
              <iconset resource="DicomSender.qrc">
               <normaloff>:/icons/send.svg</normaloff>:/icons/send.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="portEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>110</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>104</string>
             </property>
             <property name="maxLength">
              <number>5</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="placeholderText">
              <string>104</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="hostEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>110</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>localhost</string>
             </property>
             <property name="maxLength">
              <number>15</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="placeholderText">
              <string>localhost</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="local">
             <property name="text">
              <string>Local AE:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="host">
             <property name="text">
              <string>Host:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QPushButton" name="buttonEcho">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>28</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>28</height>
              </size>
             </property>
             <property name="text">
              <string>Echo</string>
             </property>
             <property name="icon">
              <iconset resource="DicomSender.qrc">
               <normaloff>:/icons/echo.svg</normaloff>:/icons/echo.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="port">
             <property name="text">
              <string>Port:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLineEdit" name="localEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>110</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>AE</string>
             </property>
             <property name="maxLength">
              <number>16</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="placeholderText">
              <string>AE</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLineEdit" name="targetEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>110</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>AE</string>
             </property>
             <property name="maxLength">
              <number>16</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="placeholderText">
              <string>AE</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="target">
             <property name="text">
              <string>Target AE:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="5">
            <widget class="QProgressBar" name="progressBar">
             <property name="value">
              <number>0</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="textVisible">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QLabel" name="status">
             <property name="text">
              <string>---</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="logEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>dirsList</tabstop>
  <tabstop>buttonOpen</tabstop>
  <tabstop>hostEdit</tabstop>
  <tabstop>portEdit</tabstop>
  <tabstop>localEdit</tabstop>
  <tabstop>targetEdit</tabstop>
  <tabstop>buttonEcho</tabstop>
  <tabstop>buttonSend</tabstop>
 </tabstops>
 <resources>
  <include location="DicomSender.qrc"/>
 </resources>
 <connections/>
</ui>
